dev.yml

```diff
@@ -8,6 +8,7 @@ up:
       gemfile: Gemfile
 
 commands:
+  docs: bundle exec rake check_docs
```

Comment:
Does `minidev` support subtasks? If so, I think `docs` should be _generating_ docs (via YARD, I believe) and `docs:check` should be for checking.

Reply:
- `dev docs` should run `bundle exec rake yard`
- `dev docs check` should run `bundle exec rake check_docs`